.github/workflows/preview.yml

```diff
@@ -0,0 +1,65 @@
+name: Expo Preview
+on:
+  push:
+    branches: [master, main]
+jobs:
+  changed:
+    name: Find changed apps
+    runs-on: ubuntu-latest
+    outputs:
+      exampleChanged: ${{ steps.filter.outputs.examples == 'true' }}
+      exampleNames: ${{ steps.find.outputs.result }}
+    steps:
+      - id: filter
+        name: 🕵️ Detect changes
+        uses: dorny/paths-filter@v2
+        with:
+          list-files: json
+          filters: |
+            examples:
```

Comment:
It's unfortunate that we can't just filter on `with-**` only. It's because we can't run the publish on things like `.gh-assets` or `.github` (or even just the root files). But, this works, and auto-picks up folder pattern `with-**`